src/cryptojwt/key_bundle.py

```diff
@@ -168,6 +168,13 @@ def wrapper(self, *args, **kwargs):
     return wrapper
 
 
+def httpc_params_loader(httpc_params):
```

Comment:
Perhaps put this in `utils.py`?

Reply:
sounds good, ok, I'm doing it right now

Reply:
done
https://github.com/IdentityPython/JWTConnect-Python-CryptoJWT/commit/703b0d861eff4ed2bfc6efc5a1c9b310ad81d1cf